db/seeds.rb

```diff
@@ -2,7 +2,7 @@
 
 def make_books
   SEEDS[:books].each do |book|
-    Book.create!  book.merge! (in_store: true)
+    Book.create!  book.merge! ({in_store: true})
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.<br>Space inside { missing.<br>Space inside } missing.